lib/node_modules/@stdlib/math/base/special/minmax/README.md

```diff
@@ -119,6 +119,102 @@ for ( i = 0; i < 100; i++ ) {
 
 <!-- /.examples -->
 
+
+<!-- C interface documentation. -->
+
+* * *
+
+<section class="c">
+
+## C APIs
+
+<!-- Section to include introductory text. Make sure to keep an empty line after the intro `section` element and another before the `/section` close. -->
+
+<section class="intro">
+
+</section>
+
+<!-- /.intro -->
+
+<!-- C usage documentation. -->
+
+<section class="usage">
+
+### Usage
+
+```c
+#include "stdlib/math/base/special/minmax.h"
+```
+
+#### stdlib_base_minmax( x, y )
+
+Returns the minimum and maximum value.
+
+```c
+double out = stdlib_base_minmax( 4.2, 3.14 );
+// returns [ 3.14, 4.2 ]
+
+out = stdlib_base_minmax( 0.0, -0.0 );
+// returns [ -0.0, 0.0 ]
+```
+
+The function accepts the following arguments:
+
+-   **x**: `[in] double` input value.
+-   **y**: `[in] double` input value.
+
+```c
+double stdlib_base_minmax( const double x, const double y );
```

Comment:
@AbhijitRaut04  This is different from the function signature that you've used in `main.c`. You might want to update this accordingly here and everywhere else.